pypeit/ginga.py

```diff
@@ -3,6 +3,7 @@
 import os
 import numpy as np
 import time
+import IPython
```

Comment:
Are we importing IPython everywhere?  I thought we would be treating these like when we were importing `pdb`.  I.e., we put it there for debugging but then remove it.  Maybe we just do this before we merge from develop into master?

Reply:
I'm slowly phasing them all out and replacing with from IPython import embed. I think it is not a big import if we just import embed.